### DISCUSSION TITLE
CSS Grid prop justify content missing several values

### DISCUSSION THREAD

**Discussion by thewebartisan7** [radix-ui/themes#457] [Q&A]
The Grid component prop justify is missing several values like stretch, normal etc... and has a default value of flex-start, which then make the grid childs not fit whole width.

Results of below example with header, sidebar and main content is in below screenshot:

```jsx
            <Grid
                rows="75px auto"
                columns="350px auto"
                // Need to override default justify content with inline style since prop justify missing it:
                style={{ justifyContent: "normal" }}
            >
                <Flex direction="column" gridRow="1" gridColumn="1/3">
                    Header
                </Flex>

                <Flex
                    gridRow="2"
                    gridColumn="1"
                    direction="column"
                >
                    Sidebar
                </Flex>

                <Flex gridRow="2" gridColumn="2" direction="column">
                    Main
                </Flex>
            </Grid>
```

Result (the yellow is the main grid:

<img width="1711" alt="Screenshot 2024-04-15 at 15 25 26" src="https://github.com/radix-ui/themes/assets/57477934/c0d9accc-aace-453f-a5b0-2b6f220a32aa">

Expected:

<img width="1723" alt="Screenshot 2024-04-15 at 15 24 54" src="https://github.com/radix-ui/themes/assets/57477934/e04e2d3c-185b-4cc6-80d6-b31a28716e95">

Would be also better to have a different default than flex-start.

**Comment by vladmoroz** [comment]
You could just do `columns=350px 1fr"`, that's more explicit and achieves what you are trying to achieve out of the box.

Not sure about changing the default, but we'll look into the missing values.

**Reply by thewebartisan7** [reply]
Ah yes, I see! Thank you. However yes adding that missing values would also be good.
